docs/how-to/rocm-for-ai/model-automation-and-dashboarding.rst

```diff
@@ -0,0 +1,154 @@
+.. meta::
+   :description: Discover, run, and benchmark deep learning models with AMD MAD -- Model Automation and Dashboarding tool.
+   :keywords: dashboard, machine, deep, container, playground, tune
+
+************************
+Running models using MAD
```

Comment:
Is this the right title?
Curious as three models talk about three things you can use MAD for.

Reply:
Your description in the metadata might be better - not sure

Reply:
Notice the MAD repo uses the following intro:
MAD (Model Automation and Dashboarding), is a combination of AI/ML model zoo, automation for running the models on various GPU architectures, a mechanism of maintaining historical performance data, and generating dashboards for tracking.

Do we need to closer align? For example, this discusses AI/ML model